module/config.mjs

```diff
@@ -163,6 +163,31 @@ ED4E.action = {
 };
 preLocalize( "action" );
 
+
+/**
+ * challengeRates
+ * @enum {string}
+ */
+ED4E.challengeRates = {
```

Comment:
I don't think this is necessary... If this is only for localization purposes just make a dynamic localization and call it with `format`, passing the circle number.
Hardcoding numbers like this is too inflexible.

Reply:
I don't understand it. lets look at it together